contracts/controller/Reputation.sol

```diff
@@ -51,8 +51,12 @@ contract Reputation is Ownable {
             balances[_to] = balances[_to].add(absAmount);
```

Comment:
@leviadam I guess that the use of safe math to protect against an overflow.right ?
So why not just test that here?.if it get to uint MAX_VALUE just set it to that..avoiding the call to safeMath.

Reply:
I think using safeMath is cleaner.